app/boards/shields/nibble/nibble.keymap

```diff
@@ -8,27 +8,40 @@
 #include <dt-bindings/zmk/keys.h>
 #include <dt-bindings/zmk/bt.h>
 
-#define DEFAULT 0
-#define FUNC 1
+#define LAYER_ID_DEFAULT 0
+#define LAYER_ID_FUNC 1
 
 / {
+    sensors {
+		compatible = "zmk,keymap-sensors";
+		sensors = <&encoder_1>;
+	};
```

Comment:
Seem to be some rogue tabs in here. Spaces are used in the rest of the file.

Reply:
Converted to tabs, sorry about that